### PR TITLE
chore(CX-2524): clean up feature flag AREnableCollectorProfile

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-2524]

### Description

Clean up feature flag AREnableCollectorProfile
### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [x] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- clean up feature flag AREnableCollectorProfile -daria

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

[CX-2524]: https://artsyproduct.atlassian.net/browse/CX-2524?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyProfile/MyProfileEditForm.tsx [hunk 1]
```diff
@@ -181,19 +178,6 @@ export const MyProfileEditForm: React.FC = () => {
...
-      // When the user applies the email verification and the modal is visible
-      if (showVerificationBannerForEmail || showVerificationBannerForID) {
-        refetch({ enableCollectorProfile })
-      }
-    }, 3000)
-
-    return () => {
-      clearInterval(refetchProfileIdentificationInterval)
-    }
-  }, [showVerificationBannerForEmail, showVerificationBannerForID])
```
Comment (dariakoko) [inline src/app/Scenes/MyProfile/MyProfileEditForm.tsx:195]: Please, let me know if it should be refactored instead 🤔

**Review by olerichter00 — CHANGES_REQUESTED** [review]
This looks good! I just think we need to keep the logic to periodically refetch the verification status.

File: src/app/Scenes/MyProfile/MyProfileEditForm.tsx (hunk 1 above)
Comment (olerichter00) [reply src/app/Scenes/MyProfile/MyProfileEditForm.tsx:195]: The query doesn't expect any arguments anymore. I guess, you can just change `refetch({ enableCollectorProfile })` to `refetch()`.

Comment (dariakoko) [reply src/app/Scenes/MyProfile/MyProfileEditForm.tsx]: Right, thank @olerichter00
```refetch({})``` because it expects an argument
Fixed :)

**Review by olerichter00 — APPROVED** [review]
🙌

**Pull request merged**